readme.md

```diff
@@ -104,6 +105,18 @@
 - [john](https://github.com/davej/john) - Make npm3's flat dependencies easier to find and sort.
 
 
+## Clients
+
+### Official
```

Comment:
No need to have separate subsection. Just note the official one in its description.